Porting of How to use example to Elixir

tomashco opened the pull request:
I added the example rewritten in Elixir for the How to use erlang example.

pguyot commented on examples/basic/ex_how_to_use/.gitignore:
```diff
@@ -0,0 +1,11 @@
+rebar.lock
+*.avm
+**.beam
+deps
```
Would you mind adding a missing new line here?

pguyot commented on examples/basic/ex_how_to_use/mix.exs:
```diff
@@ -0,0 +1,52 @@
+#
+# This file is part of AtomVM.
+#
+# Copyright 2018 Davide Bettio <davide@uninstall.it>
```
I am not sure this is the proper copyright line.

pguyot reviewed (commented):
The wav makes the file too big. Is there a way to avoid this?

For example we could put it in priv/ and use `atomvm:read_priv/2`

tomashco commented:
Hi @pguyot I moved the wav file into another module, in order to make the file lighter. I also refactored the main loop a bit in order to divide the different informations in each function. I saw also a performance improvement regarding the graphical updates in the IMU on the screen (now it's much more fluid)

I also updated the gitignore and the copyright
